nservicebus/msmq/index.md

```diff
@@ -61,4 +64,8 @@ So downtime is proportional to the time taken for the MSMQ service to restart on
 
 ## Remote Queues
 
-Remote queues are not supported for MSMQ as this conflicts with the Distributed Bus architectural style that is predicated on consents of durability, autonomy and avoiding a single point of failure. For scenarios where a Broker Bus style architecture is required use transports like [Sql Server](/nservicebus/sqlserver/) and [RabbitMQ](/nservicebus/rabbitmq/).
\ No newline at end of file
+Remote queues are not supported for MSMQ as this conflicts with the Distributed Bus architectural style that is predicated on consents of durability, autonomy and avoiding a single point of failure. For scenarios where a Broker Bus style architecture is required use transports like [Sql Server](/nservicebus/sqlserver/) and [RabbitMQ](/nservicebus/rabbitmq/).
```

Comment:
@DavidBoike - you're right, I didn't alter that section. Now that you bring that up, I don't think that's the original reason behind why remote queues aren't supported. In older versions of MSMQ, reading from remote queues wasn't possible. Although reading from remote queues is possible now, it is not performant and may requires additional permissions. Not sure who the original author is for this section @andreasohlund @SimonCropp - thoughts on this?